src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int FIX_RADIUS = 10;
+    private static final int MAX_NUMBER = 5;
+    private static final int MAX_PARAMETR = 10;
+    private static final String COLOR_WHITE = Color.WHITE.name();
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        String color = colorSupplier.getRandomColor();
+        int firstLeg = random.nextInt(MAX_PARAMETR) + 1;
+        int secondLeg = random.nextInt(MAX_PARAMETR) + 1;
+        int radius = random.nextInt(MAX_PARAMETR) + 1;
+        int height = random.nextInt(MAX_PARAMETR) + 1;

```

Comment:
create a private utility method for that, Also, why not create it right inside of cases, this way we will create only the values we will use